redfish-core/lib/pcie_slots.hpp

```diff
@@ -278,10 +278,34 @@ inline void
     });
 }
 
+inline void getLocationCode(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+                            const size_t index,
+                            const std::string& connectionName,
+                            const std::string& pcieSlotPath)
+{
+    sdbusplus::asio::getProperty<std::string>(
+        *crow::connections::systemBus, connectionName, pcieSlotPath,
+        "xyz.openbmc_project.Inventory.Decorator.LocationCode", "LocationCode",
+        [asyncResp, index](const boost::system::error_code& ec1,
+                           const std::string& property) {
+        if (ec1)
+        {
+            BMCWEB_LOG_ERROR("Can't get location code property for PCIeSlot,"
+                             "Error:{}",
```

Comment:
I didn't notice this.
This is better to be a single unbroken string - 
```
"Can't get location code property for PCIeSlot, Error:{}",
```

Reply:
Ack